src/typeahead/typeahead.js

```diff
@@ -333,6 +333,7 @@ var Typeahead = (function() {
 
     open: function open() {
       if (!this.isOpen() && !this.eventBus.before('open')) {
+        this.input.$input.attr("aria-expanded", true);
```

Comment:
I have noticed your tests are broken because $input is not available in this context, if possible create the function bellow at `input.js` and call it here to set `aria-expanded`. This will fix all the broken tests.

``` 
setAriaExpanded: function setAriaExpanded(value) {
    this.$input.attr('aria-expanded', value);
},
```